src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -325,6 +326,20 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       }
     }
 
+    const updateRecentlyViewedWorkspaces = workspaceId => {
+      const recentlyViewedPersistenceId = 'workspaces/recentlyViewed'
```

Comment:
This ID is also defined in `workspace-utils`. It would be good to avoid that duplication, and I think this code might make more sense in that file anyway.

Reply:
Moved everything to workspace-utils in ba5c1d9244b8dce9af6b7c57889cad888a153d51